index.js

```diff
@@ -39,9 +39,12 @@ module.exports = function(content) {
 	}
 	var root = config.root;
 	var links = attrParse(content, function(tag, attr) {
-		var item = tag + ":" + attr;
 		var res = attributes.find(function(a) {
-		  return item.indexOf(a) >= 0;
+			if (a.startsWith(':')) {
```

Comment:
`node >= v4` Support ? What is the alternative ? I'm working on #134 which is `v1.0.0` with min `node >= v4`. It would normally be good to stay with `node >= v0.12` in the current semver minor range, but `node >= v0.12` is EOL and eventually the next release includes #134 anyways 😛 but not :100: sure yet.

```js
const starts = (value, str) => str.substr(0, value.length) === 'value'
```